src/schedule-and-details/instructors-section/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuid } from 'uuid';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+import { Add as AddIcon } from '@edx/paragon/icons';
+
+import InstructorContainer from './instructor-container';
+import ScheduleSubHeader from '../schedule-sub-header';
+import messages from './messages';
+
+const InstructorsSection = ({ instructors, onChange }) => {
+  const intl = useIntl();
+  const newInstructor = {
+    bio: '',
+    image: '',
+    name: '',
+    organization: '',
+    title: '',
+  };
+
+  const handleChange = (value, idx, field) => {
+    const updatedInstructors = instructors.map((instructor, index) => {
+      if (index === idx) {
+        return {
+          ...instructor,
+          [field]: value,
+        };
+      }
+      return instructor;
+    });
+
+    const updatedInstructorInfo = {
+      instructors: updatedInstructors,
+    };
+    onChange(updatedInstructorInfo, 'instructorInfo');
+  };
+
+  const handleDelete = (idx) => {
+    const updatedInstructorInfo = {
+      instructors: [...instructors],
+    };
+    updatedInstructorInfo.instructors.splice(idx, 1);
+    onChange(updatedInstructorInfo, 'instructorInfo');
+  };
+
+  const handleAdd = () => {
+    const updatedInstructorInfo = {
+      instructors: [...instructors, newInstructor],
+    };
+
+    onChange(updatedInstructorInfo, 'instructorInfo');
+  };
+
+  return (
+    <section className="section-container instructors-section">
+      <ScheduleSubHeader
+        title={intl.formatMessage(messages.instructorsTitle)}
+        description={intl.formatMessage(messages.instructorsDescription)}
+      />
+      <ul className="instructors-list">
+        {instructors.map((instructor, idx) => (
+          <InstructorContainer
+            instructor={instructor}
+            key={uuid}
+            idx={idx}
+            onDelete={handleDelete}
+            onChange={handleChange}
+          />
+        ))}
+      </ul>
+      <Button iconBefore={AddIcon} variant="outline-success" onClick={handleAdd}>
```

Comment:
```suggestion
      <Button iconBefore={AddIcon} variant="primary" onClick={handleAdd}>
```

Reply:
As i understand, we have already approved style design. Do we really need to apply these changes?

Reply:
Already added)